include/ebpf_api.h

```diff
@@ -229,6 +229,7 @@ extern "C"
      * @param[out] key_size Size of keys in the eBPF map.
      * @param[out] value_size Size of values in the eBPF map.
      * @param[out] max_entries Maximum number of entries in the map.
+     * @param[out] inner_map_idx Index in the object of an inner map template.
```

Comment:
Can you clarify this? What object is this statement referring to?

Reply:
It was referring to the maps section of an ELF file, or `ebpf_object_t`

Reply:
Will "inner_map_idx" be non-zero only for the inner maps, and 0 for regular and outer maps?

Reply:
Other way around.  It is only non-zero in an outer map, pointing to a "fake" map that is the template for what inner map types can be stored in the outer map.  See the link in the PR description.

Reply:
Updated comment to clarify.